tests/functional/pod_and_daemons/test_mgr_enable_rook_backend_module.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ocs_version,
+    skipif_external_mode,
+    tier2,
+    brown_squad,
+)
+from ocs_ci.helpers.helpers import run_cmd_verify_cli_output
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import bugzilla, ManageTest
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@brown_squad
+@skipif_external_mode
+@skipif_ocs_version("<4.16")
```

Comment:
the fix was backported to previous release as well

Reply:
done